simulation_parameters/SimulationParameters.cs

```diff
@@ -495,4 +520,9 @@ private List<Compound> ComputeCloudCompounds()
     {
         return compounds.Where(p => p.Value.IsCloud).Select(p => p.Value).ToList();
     }
+
+    private List<Enzyme> ComputeHydrolyticEnzymes()
+    {
+        return enzymes.Where(e => e.Value.Property == "hydrolytic").Select(e => e.Value).ToList();
```

Comment:
Should the property be an enum type? This code makes me think that it should be otherwise someone at some point will mess up the name or check somewhere and cause a bug.